streams/src/test/java/org/apache/kafka/streams/kstream/internals/KStreamBranchTest.java

```diff
@@ -59,7 +59,7 @@ public void testKStreamBranch() {
 
         assertEquals(3, branches.length);
 
-        final MockProcessorSupplier<Integer, String> supplier = new MockProcessorSupplier<>();
+        final MockProcessorSupplier<Integer, String, Void, Void> supplier = new MockProcessorSupplier<>();
```

Comment:
Seems this changes are not related to this PR (similar below)? Would be good to exclude them and put into it's own PR.

Reply:
Yeah, problem is that I need them to decently test the topology.

Let me see what I can do